man/fread.Rd

```diff
@@ -37,7 +37,9 @@ yaml=FALSE, autostart=NA, tmpdir=tempdir()
   \item{nrows}{ The maximum number of rows to read. Unlike \code{read.table}, you do not need to set this to an estimate of the number of rows in the file for better speed because that is already automatically determined by \code{fread} almost instantly using the large sample of lines. `nrows=0` returns the column names and typed empty columns determined by the large sample; useful for a dry run of a large file or to quickly check format consistency of a set of files before starting to read any of them. }
   \item{header}{ Does the first data line contain column names? Defaults according to whether every non-empty field on the first data line is type character. If so, or TRUE is supplied, any empty column names are given a default name. }
   \item{na.strings}{ A character vector of strings which are to be interpreted as \code{NA} values. By default, \code{",,"} for columns of all types, including type `character` is read as \code{NA} for consistency. \code{,"",} is unambiguous and read as an empty string. To read \code{,NA,} as \code{NA}, set \code{na.strings="NA"}. To read \code{,,} as blank string \code{""}, set \code{na.strings=NULL}. When they occur in the file, the strings in \code{na.strings} should not appear quoted since that is how the string literal \code{,"NA",} is distinguished from \code{,NA,}, for example, when \code{na.strings="NA"}. }
-  \item{stringsAsFactors}{ Convert all character columns to factors? }
+  \item{stringsAsFactors}{ Convert all character columns to factors?
+  The default behaviour for character columns is if \code{uniqueN(column)/nrow < 0.2} then make the column a factor. Acceptable inputs are TRUE, FALSE, or a decimal value between 0.0 and 1.0. \code{stringsAsFactors = FALSE} all character columns are character. \code{stringsAsFactors = TRUE} makes all character columns factor. \code{stringsAsFactors = 0.2} makes character columns factors if \code{uniqueN(column)/nrow < 0.2}. \code{stringsAsFactors = 0.5} makes character columns factors if \code{uniqueN(column)/nrow < 0.5}. 
```

Comment:
i would prefer if this is a little more concise, I'm not sure if the behavior is intricate enough to warrant 6 sentences. i think the entry should start with the "Acceptable inputs" sentence, then maybe just refer to a generic decimal value instead of .2 and .5?